examples/predator_prey.jl

```diff
@@ -244,17 +245,13 @@ results, _ = run!(model, agent_step!, n_steps; adata)
 # The plot shows the population dynamics over time. Initially, wolves become extinct because they
 # consume the sheep too quickly. The few remaining sheep reproduce and gradually reach an
 # equilibrium that can be supported by the amount of available grass.
-
-@df results plot(
-    :step,
-    :count_sheep,
-    grid = false,
-    xlabel = "Step",
-    ylabel = "Population",
-    label = "Sheep",
-)
-@df results plot!(:step, :count_wolves, label = "Wolves")
-@df results plot!(:step, :count_grass, label = "Grass")
```

Comment:
An additional note for us devs: we can also remove the StatsPlots dependency in the docs with this change.

Reply:
ok